### DISCUSSION TITLE
The BUILD_URL details from Azure DevOps is not exposed as the expected hyperlink format

### DISCUSSION THREAD

**Discussion by sadeesh-sdet** [test-results-reporter/reporter#109] [Q&A]
To expose the BUILD_URL details from Azure DevOps as a hyperlink, I followed this [CI example](https://test-results-reporter.github.io/examples/ci.html#azure-devops-pipelines) but currently, it is not shown as hyperlink format. I followed [this another approach](https://stackoverflow.com/a/65956574) of passing values too but the results are same as below. Please advice.

![image](https://user-images.githubusercontent.com/87024075/204991388-6c7bdc20-920c-4d1d-8205-e7892ca750b9.png)

**Notes:**
1. Test Results type: Junit
2. When I pass the Hardcoded URL string, it works as intended.

**Comment by ASaiAnudeep** [comment]
Could you please share your config file.

**Reply by sadeesh-sdet** [reply]
This is my config file 
```json
{
  "reports": [
    {
      "targets": [
        {
          "name": "teams",
          "inputs": {
            "url": "actual teams-webhook url to be passed",
            "publish": "test-summary-slim"
          },
          "extensions": [
            {
              "name": "hyperlinks",
              "inputs": {
                "links": [
                  {
                    "text": "Build Logs",
                    "url": "{BUILD_URL}"
                  }
                ]
              }
            }
          ]
        }
      ],
      "results": [
        {
          "type": "junit",
          "files": ["./cypress/reports/junit/results*.xml"]
        }
      ]
    }
  ]
}
```

**Note:**To handle this, Currently exposing the BUILD_URL by passing the BUILD_ID environmental variable from CI as like below in config.js file and which works as intended.
 "url": `https://dev.azure.com/XXXX/XXXX_build/results?buildId=${BUILD_ID}`

**Reply by ASaiAnudeep** [reply]
Not able to replicate the issue. Can you make sure you are using the latest version of the tool.

**Reply by sadeesh-sdet** [reply]
@ASaiAnudeep I'm sorry for the delaying response and I have been using the latest version only.
